docs/concepts/how-celestia-works/data-availability-layer.md

```diff
@@ -87,8 +87,8 @@ As a solution, _Fraud Proofs of Incorrectly Generated Extended Data_ enable
 light nodes to reject blocks with invalid extended data. Such proofs require
 reconstructing the encoding and verifying the mismatch. With standard Reed-Solomon
 encoding, this entails downloading the original data, i.e., O(n) bytes.
-Contrastingly, with 2-dimensional Reed-Solomon encoding, only O(n ) bytes are
-required as it is sufficient to verify only one row or one column of the
+Contrastingly, with 2-dimensional Reed-Solomon encoding, only O(sqrt(n)) bytes
```

Comment:
[for reviewers] Given original data of `n` bytes which is split into a data square of `k * k` (so `k = sqrt(n)`). One column of the square is `sqrt(n) * 1`.